CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 All notable changes to this project will be documented in this file.
 
 
+#### v2.0.0 - ?
```

Comment:
According to the latest version of [Semantic Version](http://semver.org/) we shouldn't prepend the version with a `v`.